aleph-client/src/aleph_zero.rs

```diff
@@ -19533,9 +19532,9 @@ pub mod api {
         let runtime_metadata_hash = client.metadata().metadata_hash(&PALLETS);
         if runtime_metadata_hash
             != [
-                155u8, 236u8, 207u8, 138u8, 149u8, 114u8, 96u8, 168u8, 143u8, 247u8, 148u8, 106u8,
-                148u8, 203u8, 48u8, 129u8, 14u8, 48u8, 155u8, 234u8, 78u8, 212u8, 73u8, 65u8,
-                212u8, 201u8, 174u8, 194u8, 72u8, 70u8, 240u8, 233u8,
+                10u8, 121u8, 157u8, 11u8, 147u8, 107u8, 235u8, 73u8, 90u8, 254u8, 82u8, 183u8,
+                112u8, 64u8, 213u8, 99u8, 23u8, 17u8, 10u8, 91u8, 124u8, 231u8, 209u8, 172u8, 59u8,
+                160u8, 15u8, 142u8, 149u8, 200u8, 95u8, 164u8,
```

Comment:
Why is this PR passing CI if it's changing the `aleph_zero.rs`?

Reply:
It turned out sb forgot already to update `aleph_zero.rs` on `main`, hence I needed to fix it there.